index.js

```diff
@@ -0,0 +1,4 @@
+aaa = bbb;
+bbb = ccc;
+ccc = ddd;
+ddd = eee;
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'ddd' is not defined  no-undef